Podfile

```diff
@@ -1,8 +1,11 @@
+source 'https://github.com/CocoaPods/Specs.git'
+source 'https://github.com/foreseecode/podspecs'
+
 platform :ios, '10.0'
 workspace 'ForeSeeSamples.xcworkspace'
 
 def foresee_pods
-    pod 'ForeSee', '5.1.0'
+    pod 'ForeSee-test', '5.2.0-RC19'
```

Comment:
We don't want to merge this to master, right?

Reply:
Yep. See above.